lib/shippingwrapper.rb

```diff
@@ -0,0 +1,46 @@
+require 'active_shipping'
+
+class ShippingWrapper
+  attr_reader :fedex, :ups, :response
+  def initialize(fedex=nil, ups=nil, response=false)
+    if fedex.present? && ups.present?
```

Comment:
Here you don't need the .present? method if you don't want it. You can just say if fedex? Since fedex is nil that would evaluate the same. The .present? method makes it very clear though, so whatever you want!